source/lib-intro.tex

```diff
@@ -1061,55 +1061,44 @@
 
 
 \pnum
-The facilities of the C standard library are provided in 26
-\indextext{library!C standard}%
-additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
+The facilities of the C standard library\indextext{library!C standard}
+are provided in 22 additional headers, as shown in Table~\ref{tab:cpp.c.headers}.%
```

Comment:
I don't think we have a blanket rule for this. In the present case, it was easy enough to update the number, and we've previously updated (but kept) counts of things.